### ISSUE TITLE
Result-based navigation fails

### ISSUE THREAD

**Issue by swflint** [dajva/rg.el#154]
Navigation using `n` and `p` (or any `next-error` or `previous-error` variants) fails to pop up correct file, I am prompted with `Find this error in (default 1): `.

Configuration is same as #153, additionally, I am using emacs version 28.2.

**Comment by swflint** [comment]
Well, that was exactly it.  It may be worth mentioning this issue somewhere in documentation!

**Comment by dajva** [comment]
Yes, I though it was there actually. But I obviously have not got around to it yet.
